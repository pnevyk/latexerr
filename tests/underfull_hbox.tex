\documentclass{article}

\begin{document}

Donec nec sapien scelerisque, sagittis augue dictum, interdum nisl. \\ \\ Aenean est libero, porttitor vitae mi non, sodales mattis lectus.

\end{document}
