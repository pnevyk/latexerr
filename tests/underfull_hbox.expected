File: underfull_hbox.tex

Warning on line 5: Line cannot be stretch enough. The problem is very bad.
